internal/cli/terraform.go

```diff
@@ -124,10 +124,14 @@ func generateTerraformCmd(cli *cli) *cobra.Command {
 		Use:     "generate",
 		Aliases: []string{"gen", "export"}, // Reconsider aliases and command name before releasing.
 		Short:   "Generate terraform configuration for your Auth0 Tenant",
-		Long: "This command is designed to streamline the process of generating Terraform configuration files for " +
+		Long: "(Experimental) This command is designed to streamline the process of generating Terraform configuration files for " +
 			"your Auth0 resources, serving as a bridge between the two.\n\nIt automatically scans your Auth0 Tenant " +
-			"and compiles a set of Terraform configuration files based on the existing resources and configurations." +
-			"\n\nThe generated Terraform files are written in HashiCorp Configuration Language (HCL).",
+			"and compiles a set of Terraform configuration files (HCL) based on the existing resources and configurations." +
+			"\n\nRefer to the [instructional guide](https://registry.terraform.io/providers/auth0/auth0/latest/docs/guides/generate_terraform_config) for specific details on how to use this command." +
+			"\n\n**Note:** This command is experimental and is subject to change in future versions.",
```

Comment:
Should the fact that this command is experimental and subject to change in future versions be more prominent? E.g. using a "Warning:" instead of "Note:" here, and printing an actual warning when the command runs.

Reply:
Fair point. I changed the wording here from "note" to "warning". Re: experimental messaging in output, I think it would be a bit heavy-handed IMO. And by comparison, Terraform's own CLI does not do that for their own experimental features.